tests/core_tests.py

```diff
@@ -981,6 +981,43 @@ def test_import_csv(self):
             os.remove(f1)
             os.remove(f2)
 
+    def test_import_csv_filesize_limit(self):
+        max_bytes = app.config.get("UPLOAD_MAX_BYTES")
+        if not max_bytes:
+            return
+        self.login(username="admin")
+        table_name = "".join(random.choice(string.ascii_lowercase) for _ in range(5))
+        f1 = "testSize.csv"
+        self.enable_csv_upload(utils.get_example_database())
+        repeat = max_bytes // 4
+        data = "a,b"
+        content = [data for _ in range(repeat)]
+        self.create_sample_csvfile(f1, content)
```

Comment:
The single line content of the file is a,b\n.The size is 4byte.
repeat is the number of file lines.
file f1 less than or equal to UPLOAD_MAX_BYTES